frontend/constants/serviceTemplates.ts

```diff
@@ -7,20 +7,21 @@ export const SERVICE_TEMPLATES: ServiceTemplate[] = [
     description: 'Trader agent for omen prediction markets',
     image:
       'https://operate.olas.network/_next/image?url=%2Fimages%2Fprediction-agent.png&w=3840&q=75',
-    configuration: {
-      trader_version: 'v0.16.4',
-      nft: 'bafybeig64atqaladigoc3ds4arltdu63wkdrk3gesjfvnfdmz35amv7faq',
-      agent_id: 14,
-      threshold: 1,
-      use_staking: true,
-      cost_of_bond: 10000000000000000,
-      olas_cost_of_bond: 10000000000000000000,
-      olas_required_to_stake: 10000000000000000000,
-      monthly_gas_estimate: 10000000000000000000,
-      fund_requirements: {
-        agent: 100000000000000000,
-        safe: 5000000000000000000,
-      },
+    service_version: 'v0.18.1',
+    home_chain_id: '100',
+    configurations: {
+      '100': {
+        nft: 'bafybeig64atqaladigoc3ds4arltdu63wkdrk3gesjfvnfdmz35amv7faq',
+        agent_id: 14,
+        threshold: 1,
+        use_staking: true,
+        cost_of_bond: 10000000000000000,
```

Comment:
Shall this be updated to reflect Pearl Beta? The NFT is different, I suppose, so is the agent_id that must be 25, the cost_od_bond must be 20 OLAS

Reply:
Agent ID and cost of bonds, etc, are read from the staking contract. Hardcoded values only are meaningful for non-staked services.